specs/ceph_crud_flows.adoc

```diff
@@ -0,0 +1,557 @@
+// vim: tw=79
+
+= CRUD flows for ceph cluster entities
+
+The ceph-integration module need to provide flows for creating, retrieving,
+updating and deleting various entities like pools, rbds, ec-pools etc.
```

Comment:
Snapshots is not part of this spec. Only pool/rbds/ec-profiles are taken care as part of this spec.